### PR TITLE
requirements

### PR DESCRIPTION
**Pull request by RubenLech**

### CODE REVIEW

File: docs/1-introduction/1-requirements/README.md [hunk 1]
```diff
@@ -360,11 +1444,8 @@ https://github.com/YetiForceCompany/YetiForceCRM/blob/stable/.user.ini
 
 ```ini
 connect_timeout = 3600
-connect_timeout = 3600
-net_read_timeout = 3600
```
Comment (mariuszkrzaczkowski) [inline docs/1-introduction/1-requirements/README.md:364]: ?

File: docs/1-introduction/1-requirements/README.md [hunk 2]
```diff
@@ -208,16 +299,980 @@ catch_workers_output = yes
...
+;# Best FastCGI Process Manager configuration for YetiForceCRM
+;####
+
+php_admin_value[error_log] = /var/log/php/fpm_yfprod_error.log
+php_admin_value[open_basedir] = /home/yfprod/:/tmp/:/var/tmp/:/etc/nginx/ssl/:/etc/ssl/:/backup/yfprod/:/usr/bin/gpg:/usr/bin/gpg-agent:/usr/bin/gpgconf
+clear_env = no
+request_terminate_timeout = 600
+pm.process_idle_timeout = 600s;
+pm.max_requests = 5000
+catch_workers_output = yes
```
Comment (rskrzypczak) [inline docs/1-introduction/1-requirements/README.md:757]:
```suggestion
allow_url_fopen = "On"
allow_url_include = "Off"
auto_detect_line_endings = "On"
default_charset = "UTF-8"
default_socket_timeout = 600
disable_functions = pcntl_alarm,pcntl_fork,pcntl_waitpid,pcntl_wait,pcntl_wifexited,pcntl_wifstopped,pcntl_wifsignaled,pcntl_wifcontinued,pcntl_wexitstatus,pcntl_wtermsig,pcntl_wstopsig,pcntl_signal,pcntl_signal_get_handler,pcntl_signal_dispatch,pcntl_get_last_error,pcntl_strerror,pcntl_sigprocmask,pcntl_sigwaitinfo,pcntl_sigtimedwait,pcntl_exec,pcntl_getpriority,pcntl_setpriority,pcntl_async_signals,pcntl_unshare,shell_exec,exec,system,passthru,popen
display_errors = Off
display_startup_errors = Off
error_log = /var/log/php_error.log
error_reporting = E_ALL & ~E_NOTICE
expose_php = "Off"
file_uploads = "On"
html_errors = "On"
log_errors = "On"
max_execution_time = 600
max_input_time = 600
max_input_vars = 10000
mbstring.func_overload = 0
memory_limit = 1024M
mysqlnd.collect_memory_statistics = "Off"
mysqlnd.collect_statistics = "Off"
opcache.enable = "On"
opcache.enable_cli = "On"
opcache.fast_shutdown = 1
opcache.file_update_protection = 0
opcache.interned_strings_buffer = 100
opcache.max_accelerated_files = 40000
opcache.memory_consumption = 256
opcache.revalidate_freq = 0
opcache.save_comments = 0
opcache.validate_timestamps = 1
openssl.cafile=/etc/ssl/certs/ca-certificates.crt
openssl.capath=/etc/ssl/certs/
output_buffering = "On"
post_max_size = 100M
realpath_cache_size = 4096k
realpath_cache_ttl = 600
request_order = "GP"
session.auto_start = "Off"
session.cookie_httponly = "On"
session.gc_divisor = 500
session.gc_maxlifetime = 1440
session.gc_probability = 1
session.name = "YTSID"
session.cookie_secure=On
session.use_only_cookies = "On"
session.use_strict_mode = "On"
session.use_trans_sid = "Off"
short_open_tag = "On"
upload_max_filesize = 100M
user_ini.filename =
variables_order = "GPCS"
zlib.output_compression = "Off"
```

File: docs/1-introduction/1-requirements/README.md [hunk 3]
```diff
@@ -208,16 +299,980 @@ catch_workers_output = yes
...
+```
+</details>
 
-- Latest stable version: https://github.com/YetiForceCompany/YetiForceCRM/blob/stable/tests/setup/fpm/www.conf
-- Development version: https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/tests/setup/fpm/www.conf
+<details>
+  <summary>Development version</summary>
```
Comment (rskrzypczak) [inline docs/1-introduction/1-requirements/README.md:762]: remove `Development version`

File: i18n/pl/docusaurus-plugin-content-docs/current/1-introduction/1-requirements/README.md [hunk 4]
```diff
@@ -86,15 +83,15 @@ Ze względów bezpieczeństwa zalecamy aby każda baza danych posiadała dedykow
...
 [mysql]
-default-character-set           = utf8
+default-character-set           = utf8mb4
 
 [mysqld]
 default_storage_engine          = InnoDB
-default-character-set           = utf8
-character-set-server            = utf8
+default-character-set           = utf8mb4
+character-set-server            = utf8mb4
```
Comment (rskrzypczak) [inline i18n/pl/docusaurus-plugin-content-docs/current/1-introduction/1-requirements/README.md:94]: add correction from EN to PL

File: i18n/pl/docusaurus-plugin-content-docs/current/1-introduction/1-requirements/README.md [hunk 5]
```diff
@@ -208,16 +299,981 @@ catch_workers_output = yes
...
+	Options -Indexes -MultiViews
+</IfModule>
+
+```
+</details>
+<details>
+  <summary>Nginx</summary>
+
+```ini
+https://github.com/YetiForceCompany/YetiForceCRM/blob/stable/tests/setup/nginx
```
Comment (rskrzypczak) [inline i18n/pl/docusaurus-plugin-content-docs/current/1-introduction/1-requirements/README.md:1274]: link?

**Pull request closed without merging**